src/views/LoginSuccess.vue

```diff
@@ -165,7 +183,9 @@ export default class LoginSuccess extends Vue {
                 // It relieves us from having to calulate the correct array index with an
                 // unrelated counter variable.
                 pathsToDerive[this.lastDerivedIndex],
-                this.defaultAccountLabel,
+                (this.accounts.has(userFriendlyAddress))
+                    ? this.accounts.get(userFriendlyAddress)!.label
```

Comment:
Not sure, but there was a recent change to keys having this legacyAccount property now. Does it also apply here? If not, didn't we agree to use humanReadableAdress everywhere we can?

Reply:
The `legacyAccount` property is only relevant when migrating legacy account storage to new key storage. Thus not applicable here, as the Keyguard has no knowledge about account labels after the migration anymore.

Do you want to refactor `userFriendlyAddress => humanReadableAddress`? I don't remember the decision, but you may create an issue for it to track it ;).

Reply:
alright